pyproject.toml

```diff
@@ -16,6 +16,7 @@ dependencies = [
   "crcmod",
   "tqdm",
   "pandacan@git+https://github.com/commaai/panda.git@master",
+  "setuptools",
```

Comment:
make sure to leave a comment why this is is necessary. afaik this is to fix an issue with one of our dependencies